component/core/src/main/resources/db/changelog/social-rdbms.db.changelog-1.0.0.xml

```diff
@@ -888,5 +888,9 @@
     <changeSet author="social" id="1.0.0-88" dbms="hsqldb">
         <createSequence sequenceName="SEQ_SOC_LABELS_ID" startValue="1"/>
     </changeSet>
+    <changeSet author="social" id="1.0.0-89" dbms="hsqldb,oracle,postgresql">
```

Comment:
No need to add hsqldb as sequences are already added for that DB, and we do no more support Oracle

Reply:
fixed